doc/esmvalcore/preprocessor.rst

```diff
@@ -138,344 +260,559 @@ losing generality. To mask ice out, `mask_landseaice` can be used:
+.. _volume operations:
 
-The volume operations module contains the following preprocessor functions:
+Volume manipulation
+===================
+The ``_volume.py`` module contains the following preprocessor functions:
 
-| `1. extract_volume`_: extract a specific depth range from a cube.
-| `2. volume_statistics`_: calculate the volume-weighted average.
-| `3. depth_integration`_: integrate over the depth dimension.
-| `4. extract_transect`_: extract data along a line of constant latitude or
-      longitude. 
-| `5. extract_trajectory`_: extract data along a specified trajectory.
+* ``extract_volume``: Extract a specific depth range from a cube.
+* ``volume_statistics``: Calculate the volume-weighted average.
+* ``depth_integration``: Integrate over the depth dimension.
+* ``extract_transect``: Extract data along a line of constant latitude or
+  longitude.
+* ``extract_trajectory``: Extract data along a specified trajectory.
 
-1. extract_volume
------------------
 
-This function extracts a specific range in the z-direction from a cube.
-This function takes two arguments:
+``extract_volume``
+------------------
 
-* z_min: minimum in the z direction
-* z_max: maximum in the z direction
+Extract a specific range in the `z`-direction from a cube.  This function
+takes two arguments, a minimum and a maximum (``z_min`` and ``z_max``,
+respectively) in the `z`-direction.
 
-Note that this requires the requested z-coordinate range to be the same sign as
-the Iris cube, i.e. if the cube has z-coordinate as negative, then z_min and
-z_max need to be negative numbers.
+Note that this requires the requested `z`-coordinate range to be the same sign
+as the Iris cube. ie, if the cube has `z`-coordinate as negative, then
+``z_min`` and ``z_max`` need to be negative numbers.
 
 See also :func:`esmvalcore.preprocessor.extract_volume`.
 
-2. volume_statistics
---------------------
+
+``volume_statistics``
+---------------------
 
 This function calculates the volume-weighted average across three dimensions,
 but maintains the time dimension.
 
-This function takes one argument:
-* operator: operation to apply over the volume (at the moment only mean is implemented)
+This function takes the argument: ``operator``, which defines the operation to
+apply over the volume. 
 
-No depth coordinate is required as this is determined by Iris. This
-function works best when the fx files provide the cell volume.
+No depth coordinate is required as this is determined by Iris. This function
+works best when the ``fx_files`` provide the cell volume.
 
 See also :func:`esmvalcore.preprocessor.volume_statistics`.
 
 
-3. depth_integration
---------------------
+``depth_integration``
+---------------------
 
-This function integrates over the depth dimension. It performs a weighted sum
-along the z-coordinate, and removes the z direction of the output cube. It takes no arguments.
+This function integrate over the depth dimension. This function does a weighted
```

Comment:
Should be "This function integrates..." instead of "This function integrate..."?

Reply:
Done.